doc/templates.rst

```diff
@@ -604,6 +604,10 @@ exist:
 
     {# keys as expressions (the expression must be enclosed into parentheses) #}
     { (1 + 1): 'foo', (a ~ 'b'): 'bar' }
+    
+    {# keys as variables #}
+    {% set dynamic_key = 'foo' %}
+    { (dynamic_key): 'foo' }
```

Comment:
That's the exact same example as the one above. What about merging the two with 3 examples: a "simple" var, an operation (1 + 1) and a concatenation (a ~ 'b')?